examples/todoapp/lib/base/di/todoapp_with_dependencies.dart

```diff
@@ -107,22 +109,28 @@ class TodoappWithDependencies extends StatelessWidget {
 
   List<SingleChildWidget> get _libs => [
         ...TranslationsDependencies.from(baseUrl: config.baseUrl).providers,
+        ...RealmDependencies().providers,
       ];
 
   List<Provider> get _dataSources => [
+        Provider<ConnectivityDataSource>(
+          create: (context) => ConnectivityDataSource(),
+        ),
+        Provider<TodoLocalDataSource>(
+          create: (context) => TodoLocalDataSource(
+            context.read<RealmService>().realm,
```

Comment:
Please register the realm instance in the DI for direct access.

Reply:
I'm not sure if it's possible, since the Realm instance requires config that in order to initialize needs to await for a future to finish.

```
    final **user** = await _realmInitRepository.getRealmUser(app);
    final **realmConfig** = Configuration.flexibleSync(**user**, [TodoModel.schema]);
    realm = Realm(**realmConfig**);
```